### DISCUSSION TITLE
Programmatically prefetch with useSWRInfinite

### DISCUSSION THREAD

**Discussion by chris-wm** [vercel/swr#966] [Q&A]
Hi, is it possible to programmatically prefetch with useSWRInfinite? I've tried using the mutation method described in the prefetching section which triggers the fetch call fine but it only create the cache keys required for standard useSWR

**Comment by koba04** [comment]
`useSWRInfinite` uses own algorithm for the cache key, so it's difficult to mutate for prefetching without depending on the internal implementation.

But I think the use-case seems to make sense.

@shuding What do you think of prefetching for `useSWRInfinite`?

**Comment by chris-wm** [comment]
So just to clarify it's currently not possible but you consider it a legitimate use case for future development?
